docker/quickstart/docker-compose-without-neo4j.quickstart.yml

```diff
@@ -5,157 +5,157 @@ services:
   broker:
```

Comment:
Are these required format changes? 

Also, these need to be generated by the script. Unfortunately, the PR for adding m1 went in without such capabilities. Can we fix the python script for generating these yaml to format correctly instead of fixing the yaml directly?